extensions/scheduler/deployment/src/test/java/io/quarkus/scheduler/test/PausedSchedulerTest.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.scheduler.test;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.Scheduler;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class PausedSchedulerTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest test = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(DisabledSchedulerTest.Jobs.class)
```

Comment:
```suggestion
                    .addClasses(PausedSchedulerTest.Jobs.class)
```

Thank you for adding a test. I think we added a wrong job class. It should be `PausedSchedulerTest #Jobs`.  Can you have a look at this, once it is done please squash the commits into one commit as suggested by @gsmet and this will be good for merging.

Reply:
thanks for review.
i've replaced with PausedSchedulerTest and squashed commits.